codespell_lib/data/dictionary.txt

```diff
@@ -16051,6 +16051,10 @@ refrenced->referenced
 refrences->references
 refrencing->referencing
 refrerence->reference
+refrerenced->referenced
+refrerenceing->referencing
+refrerences->references
+refrerencial->referencial
```

Comment:
referential not referencial

Reply:
Fixed in 5a93d9c  
Also added `referencial->referential` in 073f0a1